src/pages/index.js

```diff
@@ -140,6 +195,28 @@ function handleCardClick(data) {
   addImagePopup.open(data);
 }
 
+function handleLikeClick(card) {
+  api.handleLike(card.id, card.isLiked).then((res) => {
+    card.updateIsLiked(res.isLiked);
+  });
+}
+function handleDeleteClick(card) {
+  deletePopup.open();
+  deletePopup.setSubmitAction(() => {
+    deletePopup.setLoading(true);
+    api
+      .handleDeleteCard(card.id)
+      .then(() => {
+        card.deleteCard();
+        deletePopup.close();
+        deletePopup.setLoading(false);
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.